src/components/structures/UserSettings.js

```diff
@@ -359,6 +363,8 @@ module.exports = React.createClass({
     },
 
     _renderLabs: function () {
+        if (this.props.enableLabs === false) return null;
```

Comment:
Intention was to make it default to enabled if it wasn't specified, which I'ver clarified in a comment.